lib/vanagon/component.rb

```diff
@@ -231,13 +231,18 @@ def get_build_dir
     # Fetches secondary sources for the component. These are just dumped into the workdir currently.
     #
     # @param workdir [String] working directory to put the source into
-    def get_sources(workdir)
+    def get_sources(workdir) # rubocop:disable Metrics/AbcSize
+      # create an empty array if @extract_with doesn't already exist
+      @extract_with = Array(@extract_with)
```

Comment:
May not be an issue since we should only get_sources once, but was mostly using `Array` because `foo = ['a']; Array(foo) = ['a']` whereas `[foo] = [['a']]`